content/en/docs/setup/windows/user-guide-windows-nodes.md

```diff
@@ -220,27 +223,42 @@ Download the [Flannel start.ps1](https://github.com/Microsoft/SDN/blob/master/Ku
 ```PowerShell
 cd c:\k
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-wget https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/flannel/start.ps1 -o c:\k\start.ps1
+Invoke-WebRequest https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/flannel/start.ps1 -o c:\k\start.ps1
```

Comment:
`wget` is a PowerShell alias for `Invoke-WebRequest`. The parameters for that don't match `wget` on Linux/MacOS, so this should be less confusing.

Reply:
:)